packages/runtime/container-runtime/src/summary/orderedClientElection.ts

```diff
@@ -416,13 +416,25 @@ export class OrderedClientElection
 			change = true;
 		}
 		if (change) {
+			this.logger.sendTelemetryEvent({
+				eventName: "SummarizerClientElected",
```

Comment:
Maybe you should also add whether the parent changed or client here since any of them will set changed to true. Not sure if that info is helpful or not though.